lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -1228,10 +1231,9 @@ bool SymbolFileDWARF::ParseLineTable(CompileUnit &comp_unit) {
   if (offset == DW_INVALID_OFFSET)
     return false;
 
-  ElapsedTime elapsed(m_parse_time);
```

Comment:
We want this to stay in this function so it measures all of the work done by lldb. `m_parse_time` measures both how long it takes us to parse the information, but also should include any needed work to make this data usable for LLDB.

Reply:
My understanding is m_parse_time is the time spent only on parsing the raw data and not about constructing the data structures which will be used by lldb. In https://github.com/llvm/llvm-project/blob/b7611370491873722e08e4ce9374312d0c936af1/lldb/source/Plugins/SymbolFile/DWARF/DWARFUnit.cpp#L223, m_parse_time is used to only track the time spent on parsing a DIE in .debug_info, which also doesn't include the time on constructing necessary data structures. This change is trying to do the same by narrowing the time to just on parsing with llvm.

Reply:
We want all of the time in LLDB to parse and prepare the data, not just the parsing time. We use this same variable via `ElapsedTime elapsed(m_dwarf.GetDebugInfoParseTimeRef());` in DWARFUnit.cpp to time how long it takes to parse and prepare the DIE tree for usage.

So the m_parse_time _is_ intended to be used to measure how long LLDB needs to parse the DWARF from the object file _and_ how long it takes to prepare the data so it can be used.